content/handbook/support/providing_excellent_customer_service.md

```
@@ -0,0 +1,125 @@
+## 期待値の管理
+
+ユーザーに更新を送信するとき、通常はチケットステータスを `Pending` に設定します。これは、ユーザーからの返信を待っている状態を示します。
+
+進捗を継続的に確認するため、適切な時間が経過したら再度連絡することを顧客に期待値として伝えることも検討してください。
+
+また、チケットステータスを `On-Hold` に設定して、何らかのアクションを実行中で、後ほどユーザーにフォローアップすることを示すこともできます。
+
+チケットを `On-Hold` にする場合は次のようにしましょう:
+
+- いつフォローアップメッセージを送るかをユーザーに伝える。
+- 選択したスケジュールがニーズに合わない場合は知らせるよう促し、必要に応じて計画を調整できるようにする。[ビジネスインパクトを理解する](#understanding-business-impact)も参照してください。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)